### ISSUE TITLE
Improve sysvar documentation

### ISSUE THREAD

**Issue by nullnotnil** [pingcap/docs#3260]
## Change Request

**This repository is ONLY used to solve issues related to DOCS.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

In the PR https://github.com/pingcap/docs/pull/3152 the following comments were left unhandled:

* Has the value of `auto` been removed from `tidb_enable_table_partition`? @tiancaiamao 
* Can we safely set tidb_row_format_version to a lower value in a newer versioned TiDB cluster? @coocood
* Is there an example where `windowing_use_high_precision` has different behavior / performance? @SunRunAway 

2. Describe your suggestion or addition.

See above.

3. Provide some reference materials (documents, websites, etc) if you could.

https://github.com/pingcap/docs/pull/3152

**Comment by tiancaiamao** [comment]
`auto` is not removed from `tidb_enable_table_partition`, the semantic of `on` is changed.
I explained why in the PR https://github.com/pingcap/tidb/pull/16497
@nullnotnil

**Comment by TomShawn** [comment]
@coocood @SunRunAway PTAL

**Comment by coocood** [comment]
Yes, we can safely set tidb_row_format_version to a lower value in a newer versioned TiDB cluster.
